storybook/src/stories/Tab.stories.svelte

```diff
@@ -0,0 +1,54 @@
+<script context="module" lang="ts">
+import { Tab } from '@podman-desktop/ui-svelte';
+import { type Args, defineMeta, setTemplate, type StoryContext } from '@storybook/addon-svelte-csf';
+
+/**
+ * These are the stories for the `Tab` component.
+ * Interact with tabs.
+ */
+const { Story } = defineMeta({
+  component: Tab,
+  title: 'Tab',
+  tags: ['autodocs'],
+  args: {
+    title: 'Tab',
+    selected: false,
+  },
+  argTypes: {
+    title: {
+      control: 'text',
+      description: 'Title of the tab',
+      defaultValue: 'Tab',
+    },
+    selected: {
+      control: 'boolean',
+      description: 'Flag the tab as being selected',
+      defaultValue: false,
+    },
+  },
+});
+</script>
+
+<script lang="ts">
+setTemplate(template);
+</script>
+
+{#snippet template({ items }: Args<typeof Story>, _context: StoryContext<typeof Story>)}
+  <div class="flex flex-row px-2">
+    {#each items as item}
+      <Tab {...item} />
+    {/each}
+  </div>
+{/snippet}
+
+<Story
+  name="Basic"
+  args={{
+    items: [{ title: 'title 1' }, { title: 'title 2' }, { title: 'title 3' }],
+  }} />
+
+<Story
+  name="Selected"
+  args={{
+    items: [{ title: 'title 1', selected: false }, { title: 'title 2', selected: true }, { title: 'title 3' }],
```

Comment:
Unfortunately, the `items` value is not displayed in the code and it isnot possible for the user to see what changes. 

Could it be simpler to have a more static template, with let's say 2 tabs, and 4 params title1, selected1, title2 and selected2?

![tab-stories-items](https://github.com/user-attachments/assets/6c51ae4a-b3db-4c81-8c08-402e6a3e3c3b)

Reply:
Here I don't know how to proceed in a good way

As either it's a single tab example 

Or if it's more complex then it's not a single component but multiple components 

I can switch to the single element way

Reply:
I suppose the single Tab way would be simpler, and still useful

Reply:
switch to a single Tab